packages/table/src/TableV10/TableBody.tsx

```diff
@@ -15,7 +15,9 @@ type TableBodyProps<Shape> = Pick<TableProps<Shape>, 'children'>;
  */
 function useRenderedChildren<Datum>(
   data: Array<Datum>,
-  renderFunction: React.FunctionComponent<TableRowInterface<Datum>>,
+  renderFunction: React.FunctionComponent<
+    React.PropsWithChildren<TableRowInterface<Datum>>
```

Comment:
We should be flagging everything that updates to use `PropsWithChildren` with a patch change